src/main/java/edu/umich/tl/Course.java

```diff
@@ -36,5 +36,18 @@ public Course setEndDate(DateTime endDate) {
 		this.endDate = endDate;
 		return this;
 	}
+	
+	public static String getCourseHeader() {
+		StringBuilder courseHeader=new StringBuilder();
+		courseHeader.append("COURSE_NAME");courseHeader.append(',');
```

Comment:
For those append lines, can we get each append command in a separate line? It is easier to read in that way.